systems/plants/RigidBody.cpp

```diff
@@ -28,7 +28,9 @@ RigidBody::RigidBody(void) :
     dSdotVdvi(TWIST_SIZE, 0),
     JdotV(TWIST_SIZE, 1),
     dJdotVdq(TWIST_SIZE, 0),
-    dJdotVdv(TWIST_SIZE, 0)
+    dJdotVdv(TWIST_SIZE, 0),
+    group(DrakeCollision::DEFAULT_GROUP),
```

Comment:
should we call these collision_filter_group and collision_filter_mask to be more clear?